packages/platforms/accton/arm64/as4224/src/arm64_accton_as4224/module/src/thermali.c

```diff
@@ -98,6 +114,7 @@ onlp_thermali_info_get(onlp_oid_t id, onlp_thermal_info_t* info)
 
     /* Set the onlp_oid_hdr_t and capabilities */
     *info = linfo[tid];
-
+    info->hdr = (get_platform_id() == AS4224_48X) ? hdr_as5114[tid-1] :
```

Comment:
Please confirm the logical command here:
if (get_platform_id() == AS4224_48X) then hdr_as5114[tid-1]  ...  
you sure it is right?

Reply:
AS4224_48X is typo in the earlier development stage.
Since it is not related to thermal units, I will create another PR to fix the typo.
Thanks for your reminder.